portfolio/src/main/webapp/index.html

```diff
@@ -15,30 +15,34 @@
           <h1 id="nav-name">Sabrina Snider</h1>
         </div>
         <div id="nav-links">
-          <button class="nav-item">About Me</button>
-          <button class="nav-item">Work</button>
-          <button class="nav-item">Projects</button>
+          <a href="#about-me-section" class="nav-item">About Me</a>
+          <a href="#work-section"  class="nav-item">Work</a>
+          <a href="#project-section"  class="nav-item">Projects</a>
         </div>
       </div>
-
-      <div id="intro">
-        <h1 class="big-header-1">Hello, I'm Sabrina Snider.</h1>
+      
+      <section id="intro">
+        <h1 class="big-header-1">Hello, I'm <span style="color: #9366e3">Sabrina Snider</span>.</h1>
         <h2 class="sub-header-1">I like writing code.</h2>
-      </div>
+      </section>
 
-      <div id="smaller">
-        <div id="about-me-section" class="section white">
+      <section id="about-me-section" class="white">
+        <div class="section">
           <h1>About me</h1>
         </div>
+      </section>
 
-        <div id="work-section" class="section gray">
-          <h1>About me</h1>
+      <section id="work-section" class="gray">
+        <div class="section">
+          <h1>My Work</h1>
         </div>
+      </section>
 
-        <div id="project-section" class="section white">
-          <h1>About me</h1>
+      <section id="project-section" class="white">
+        <div class="section">
```

Comment:
Now that you're using the section tag, I think we should name this class something other than "section". Maybe "section-content" or "section-inner"?

Reply:
Good point! Changed to section-content.